backend/data_distro/api_tests/inputs/auditee_0.json

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
I don't think we need any of these `{view}_0.json` files. I believe they were mostly there for setup and I just followed the pattern since it seemed easiest!